Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/duffelbag.yml

```diff
@@ -0,0 +1,23 @@
+- type: entity
+  parent: ClothingBackpackDuffelSyndicateBundle
+  id: ClothingBackpackDuffelSyndicateFilledRailgun
+  name: Decimator bundle
+  description: "Favored by Gorlex Maurauder snipers, this railcannon packs a punch. Comes with 3 shots."
+  components:
+  - type: StorageFill
+    contents:
+      - id: WeaponRailgunDecimator
+      - id: BulletRailgunBasic
+        amount: 3
+
+- type: entity
+  parent: BoxCardboard
+  id: RailgunAmmoBag
+  name: Box of Rebar
+  description: A box for storing metal rods, commonly used in construction.
+  #prefix: Syndicate   # What is this meant to be for?
```

Comment:
```suggestion
```
Prefixes show in the entity spawner, a bit of extra info for mappers or whoever

Reply:
it's suffix :trollface:

Reply:
anyways i did this